solarforecastarbiter/datamodel.py

```diff
@@ -1081,6 +1083,7 @@ class Report(BaseModel):
     forecast_observations: Tuple[Union[ForecastObservation, ForecastAggregate],
                                  ...]
     metrics: Tuple[str, ...] = ('mae', 'mbe', 'rmse')
+    categories: Tuple[str, ...] = ('Total', 'Date', 'Hour of the day')
```

Comment:
a little odd the metrics are lower case while the categories are title case, but I'm ok going forward with this and revisiting consistency later.

Reply:
I can change them back. I think I got confused at some point that someone wanted them to be changed to match what the report shows.

Reply:
I'm not sure if it would be better for metrics to change to match categories or categories to change to match metrics. So we can leave it for now.

Reply:
ok